### PR TITLE
Show login screen in tablet at startup if not logged in and connecting to a domain that requires login

### PR DESCRIPTION
**Pull request by druiz17**

### CODE REVIEW

**Comment by howard-stearns** [comment]
Are we pushing people to log in now?
Are we requiring it? If not, how do we convey to people that it's still their choice, and we're just nagging them?

Is there a project spec that explains the business drivers for this? Who signed off on it?

**Comment by sethalves** [comment]
This is just an attempt at duplicating the behaviour in the master branch.

**Comment by hifi-gustavo** [comment]
Build console output for commit: f4255735a9fa1448af9690094ad25fca775dad89 available at: https://gist.github.com/0c6f6b8711734642d07d05815e92af7c<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10018-f425573.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10018-f425573.dmg

**Comment by howard-stearns** [comment]
I don't understand. Master does not bring up login screen at startup if not logged in.
(And if that changed in the last few hours, it still doesn't answer my questions.)

**Comment by sethalves** [comment]
Master does bring up alogin screen at startup.  The answers to your other questions are some combination of "no" and "not part of this project"

File: interface/src/ui/LoginDialog.cpp [hunk 1]
```diff
@@ -47,11 +47,11 @@ void LoginDialog::showWithSelection()
     if (tablet->getToolbarMode()) {
         LoginDialog::show();
     } else {
-        if (!hmd->getShouldShowTablet() && !qApp->isHMDMode()) {
-            LoginDialog::show();
-        } else {
-            static const QUrl url("../../dialogs/TabletLoginDialog.qml");
-            tablet->pushOntoStack(url);
+        static const QUrl url("../../dialogs/TabletLoginDialog.qml");
+        qDebug() << "[DR] -> Push login onto the tablet";
```
Comment (sethalves) [inline interface/src/ui/LoginDialog.cpp]: remove this qDebug or make it a qCDebug

**Comment by hifi-gustavo** [comment]
Build console output for commit: cba7adf32a3f0f8a3f37740f2e7306b06bc56d29 available at: https://gist.github.com/0c6f6b8711734642d07d05815e92af7c<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10018-cba7adf.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10018-cba7adf.dmg

**Pull request merged**